server/controllers/crm/crm7Controller.ts

```diff
@@ -16,6 +16,14 @@ export default class Crm7Controller {
       const usn = Number(req.params.usn)
       const { sectionId } = req.params
       const crm7Response = await this.crm7Service.getCrm(usn, getProfileAcceptedTypes(res))
+
+      const mergedScheduleCostsData = {
+        ...crm7Response.formDetails.scheduleOfTimeSpent,
+        ...crm7Response.formDetails.claimOfCosts,
+      }
+
+      crm7Response.mergedScheduleCostsData = mergedScheduleCostsData
```

Comment:
Is setting `mergedScheduleCostsData` on the crm7Response required here? As you are also passing `mergeScheduleCostsData` directly to the template.